torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1327,6 +1327,7 @@ class AliasDb::WorkingSet {
   // Add `n` to the working set
   void add(Node* n) {
     nodes_.push_back(n);
+    node_to_index_[n] = nodes_.size() - 1;
```

Comment:
This isn't quite right: here, you're setting `node_to_index_[n]` to `nodes_.size()`. `nodes_.size()` will get reconstructed each time we try `tryMove`. So you will have multiple unrelated nodes that have the same index value.

Reply:
thought node_to_index was a member of aliasDb, but it's amember of working set so we're not duplicating. my bad